core/src/main/scala/org/apache/spark/sql/delta/commands/WriteIntoDelta.scala

```diff
@@ -198,8 +198,7 @@ case class WriteIntoDelta(
       }
     }
     val rearrangeOnly = options.rearrangeOnly
-    // TODO: use `SQLConf.READ_SIDE_CHAR_PADDING` after Spark 3.4 is released.
-    val charPadding = sparkSession.conf.get("spark.sql.readSideCharPadding", "false") == "true"
+    val charPadding = sparkSession.conf.get(SQLConf.READ_SIDE_CHAR_PADDING.key, "false") == "true"
```

Comment:
The default in Spark is "true" so we cannot use `sparkSession.conf.get(SQLConf.READ_SIDE_CHAR_PADDING)`